libraries/collectd_service.rb

```diff
@@ -104,6 +104,13 @@ def action_enable
             notifies :restart, new_resource, :delayed
           end
 
+          # Installing package starts collectd service automatically
+          # Disable this so that collectd can be managed through poise-service
+          service new_resource.package_name do
+            action [:disable, :stop]
```

Comment:
@sh9189 would you be able to take a look to see if the [dpkg_autostart cookbook](https://github.com/hw-cookbooks/dpkg_autostart) does the job here? I would also be interested in knowing if there was a way to ensure that the default sysvinit/upstart scripts weren't included.

This may be something that @coderanger wants to eventually handle in the upstream poise-service?